### ISSUE TITLE
Coffee machine?

### ISSUE THREAD

**Issue by Syndaryl** [blay09/CookingForBlockheads#352]
I'm sitting here clutching my morning cup of coffee. As an "appliance" like the toaster, a coffee machine would work with harvestcraft coffee-based drinks and possibly other mods coffee-based drinks (Actually Additions' potion-like coffee would be out of scope and use AAs coffee machine as usual). 

Alternate models of a coffee percolator, filter machine, an espresso machine, and a coffee pod machine would be entertaining but that'd be a thing later as an enhancement if it appeals to you and your modeler. I'd like a kettle and coffee press but I'm not sure how common that is.

A pod machine might offer the option of doing coffee *and* tea based beverages, but it seems unsatisfying to do "cooking" using a pod machine. On the other hand, people have a big attachment to their hot beverages and the rituals around them. Possibly make a kettle and tea pot for just tea?

**Comment by Krougal** [comment]
As someone who lives on coffee this is something I can get behind. The problem I can see is how would it work? The toaster is very simple and easy, it only takes bread and it only makes toast (although come to think of it, is it CfB toast or Pam's HC toast?). No matter what other mods changes are made to how bread is made, at the end of the (MC) day, (MC)bread is still (MC)bread. Coffee not being a vanilla item would require a dependency on another mods coffee, and there are several. Otherwise CfB would have to have its own coffee. I guess the answer to that is in the answer to the toast.

**Comment by BlayTheNinth** [comment]
Cooking for Blockheads doesn't add any food items such as toast or coffee of its own, so if this were to be implemented, it would use coffee beans and coffee cups from whatever other mods are installed similar to how toast works now.

**Comment by Krougal** [comment]
> Cooking for Blockheads doesn't add any food items such as toast or coffee of its own, so if this were to be implemented, it would use coffee beans and coffee cups from whatever other mods are installed similar to how toast works now.

So if this was your only mod, the toaster would do nothing?

**Comment by BlayTheNinth** [comment]
Added in Minecraft 26.1, but it does not have functionality yet. Waiting on more mods to port to 26.1 so I can get an overview of what kind of coffee recipes exist.
